dockerfiles/docker-compose.yml

```diff
@@ -144,6 +146,7 @@ services:
       "--statediff.watchedaddresses", "0x64DE91F5A373Cd4c28de3600cB34C7C6cE410C85", # CONTRACT_MEDIAN_ETH_ADDRESS
       "--statediff.watchedaddresses", "0x83076a2F42dc1925537165045c9FDe9A4B71AD97", # CONTRACT_MEDIAN_KNC_ADDRESS
       "--statediff.watchedaddresses", "0x681c4F8f69cF68852BAd092086ffEaB31F5B812c", # CONTRACT_MEDIAN_MANA_ADDRESS
```

Comment:
Does FLIP_PAXUSD_A_1.1.1_ADDRESS also need to be down here?

Reply:
I think that's on line 120